testing/behavior_tests.bats

```diff
@@ -3,24 +3,23 @@
 # ~~~~~~~~~~~~~~~~~~~~~~
 #
 # Requirements:
-# - Kerberos principal for the iRODS service account (e.g. rods@REALM).
-# - NFSRODS Docker test container.
-# - NFSRODS must be mounted at /mnt/nfsrods.
+#   - iRODS v4.2.9+ server
+#   - NFSRODS must be mounted at /mnt/nfsrods.
+#   - GNU parallel
 #
 # How to run the tests:
-# $ docker run -d --rm --name irods -h irods_test irods_ub16_postgres
-# $ Configure NFSRODS to point to the container.
-# $ kinit rods
-# $ sudo mount -o sec=krb5,port=2050 <hostname>:/ <mount_point>
-# $ bats behaviour_tests.bat
+#   1. Configure NFSRODS to point to an iRODS server.
+#   2. Mount the NFSRODS 
+#   3. Set the home collection (e.g. export NFSRODS_TEST_USER=rods)
+#   4. bats behaviour_tests.bat
 #
 
 setup() {
     cd /mnt/nfsrods/home/rods
 }
 
 @test "test list contents of collection" {
-    ls /mnt/nfsrods/home/rods
+    /bin/ls /mnt/nfsrods/home/rods
```

Comment:
not essential.... consider defining an `$LS_EXECUTABLE` and using that everywhere in this file